backend/main.development.js

```diff
@@ -28,22 +28,14 @@ app.on('ready', () => {
         height: 1024
     });
 
-    const URL = `${server.protocol}://${server.domain}`;
-    const SETTINGS = join('&',
-        [
-            'APP_DIRECTORY',
-            'PLOTLY_API_DOMAIN',
-            'PORT'
-        ].map(s => {
-            return `${s}=${getSetting(s)}`;
-    }));
+    const {httpServer, httpsServer} = server;
+    const HTTP_URL = `${httpServer.protocol}://${httpServer.domain}:${httpServer.port}`;
+    const HTTPS_URL = `${httpsServer.protocol}://${httpsServer.domain}:${httpsServer.port}`;
 
-    console.log(`Visit ${URL}:${getSetting('PORT')}`);
-    // TODO - Does this work too?
-    // mainWindow.loadURL(`http://localhost:${getSetting('PORT')}`);
+    console.log(`Visit ${HTTP_URL}`);
 
     // Provide the port of the server to the front-end as a query string param.
-    mainWindow.loadURL(`file://${__dirname}/../static/setup.html?URL=${URL}&${SETTINGS}`);
+    mainWindow.loadURL(`${HTTP_URL}/setup`);
```

Comment:
so this works inside the electron environment? or does this open up a web browser? I feel like I tried this at one point and it didn't work